### PR TITLE
NextJS Migration

### PR DESCRIPTION
**Pull request by oetherington**

### CODE REVIEW

**Comment by MnsrIl** [comment]
Excellent work!

File: components/Navbar/MobileMenu.js [hunk 1]
```diff
@@ -18,10 +18,9 @@ const MobileMenu = ({ t }) => {
         <div className="w-full">
           <CloseIcon
             alt="Close menu"
-            width={35}
-            height={35}
             onClick={toggleMenu}
-            className="cursor-pointer float-right mt-4 mr-4 fill-yellow-ukraine"
+            style={{ transform: "scale(1.3)" }}
+            className="cursor-pointer float-right mt-8 mr-8 fill-yellow-ukraine"
```
Comment (MnsrIl) [inline components/Navbar/MobileMenu.js:23]: Can't we replace the style above to class ``scale-[1.3]``?

File: components/Navbar/MobileMenuButton.js [hunk 2]
```diff
@@ -7,9 +7,10 @@ const MobileMenuButton = () => {
   return (
     <MenuIcon
       alt="Open menu"
-      width={35}
+      width={40}
       onClick={toggleMenu}
-      className="cursor-pointer mr-4 ml-4 h-9 fill-yellow-ukraine lg:hidden"
+      style={{ transform: "scale(1.3)" }}
+      className="cursor-pointer mr-4 ml-4 mt-4 h-9 fill-yellow-ukraine lg:hidden"
```
Comment (MnsrIl) [inline components/Navbar/MobileMenuButton.js:13]: The same question, like above is here

File: next-i18next.config.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const IS_DEVELOPMENT = process.env.NODE_ENV === 'development';
```
Comment (MnsrIl) [inline next-i18next.config.js:1]: We already have this kind of constant in [configs/constants.js](https://github.com/Ukraine-Relief-Efforts/ukraine-info-guide/blob/4443bfab938507161d3b9d1d0e24eb217a54ee5e/configs/constants.js)

File: components/Navbar/MobileMenu.js (hunk 1 above)
Comment (oetherington) [reply components/Navbar/MobileMenu.js:23]: Nicely spotted - I've not used tailwind before so I often miss things like this!

File: next-i18next.config.js (hunk 3 above)
Comment (oetherington) [reply next-i18next.config.js:1]: The problem is that [configs/constants.js](https://github.com/Ukraine-Relief-Efforts/ukraine-info-guide/blob/4443bfab938507161d3b9d1d0e24eb217a54ee5e/configs/constants.js) (like all of our code) is a JS module, but in this config file we can only use `require()`/`module.exports` and not `import`/`export`.

I'm going to go ahead and merge this now, but if you've got a better solution please feel free to make a ticket for it!

Thanks for the review!

**Pull request merged**